[OSDOCS#17176]: Make changes as per the errors and warnings for CQA-3 (#114266)
openshift/openshift-docs: security/container_security/security-hosts-vms.adoc

toc::[]

[role="_abstract"]

> Both containers and virtual machines provide ways of separating applications running on a host from the operating system itself. Understanding {op-system}, which is the operating system used by {product-title}, will help you see how the host systems protect containers and hosts from each other.

Reviewer: ```suggestion
[role="_abstract"]
Containers and virtual machines provide ways of separating applications running on a host from the operating system itself. You should understand {op-system}, which is the operating system used by {product-title}, to see how the host systems protect containers and hosts from each other.
```
